slackbot_using_azure_openai/service.bal

```diff
@@ -0,0 +1,82 @@
+import ballerina/crypto;
+import ballerina/http;
+import ballerina/url;
+import ballerinax/azure_openai.chat;
+
+configurable string openAIToken = ?;
+configurable string serviceUrl = ?;
+configurable string deploymentId = ?;
+configurable string slackSigningSecret = ?;
+
+const API_VERSION = "2023-03-15-preview";
+
+final chat:Client azureOpenAI = check new (
+    config = {httpVersion: http:HTTP_1_1, auth: {apiKey: openAIToken}},
```

Comment:
You no longer need to specify the httpVersion.

Reply:
Addressed in [8b414ef](https://github.com/ballerina-guides/ai-samples/pull/64/commits/8b414efd4c7f6c79f5e6913b45a946ab677e04f3)